lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -6243,63 +6243,66 @@ namespace Js
             {
                 if (typeHandler->IsSharable())
                 {
+                    bool cachedProtoCanBeCached = nullptr;
+                    if (type->GetPrototype() == JavascriptOperators::GetPrototypeObjectForConstructorCache(constructor, requestContext, cachedProtoCanBeCached)
```

Comment:
Would that shut cases with 'overridingNewTarget' completely out of the constructor cache, even if the cache is not being used in the first place?

---
In reply to: [127739790](https://github.com/Microsoft/ChakraCore/pull/3328#discussion_r127739790) [](ancestors = 127739790)